docs/about-the-consoles/diamond.md

```diff
@@ -0,0 +1,140 @@
+current wheel functions at the bottom of the right hand screen.
+
+\(B\) The **Playback faders** and **Rotary playback faders**
+ select and control cues, programs and other configurable functions.
+
+\(C\) The **System Prompt and Menu Softkeys** have their own screen.
+
+\(D\) The **Programming and setup controls** configure and program the console,
+within which the **attribute touchscreen** shows and sets fixture attributes and 
+the **macro/executor buttons** allow storage of frequent key
+sequences, screen layouts, groups or programs. To the left of these
+controls are the cue list playback controls.
+
+\(E\) Three **Preview screens** are provided for monitoring
+video feeds or can be used as extra touch displays.
+
+
+
+## Touch screen controls
+
+![Diamond-F-G](/docs/images/Diamond-F.png)
+
+\(F\) The **Workspace windows** on the touch screens contain touch buttons for
+fixtures, palettes, groups, playbacks, macros and more in a user-configurable
+layout. You can also set fixture attributes and show information windows.
+
+\(G\) The **System Prompt and Menu Softkeys** screen is the nerve centre of
+the console and shows you what is going on. This display contains touch
+buttons for the main menu which change depending on what the console
+is doing. Softkey commands are shown in the manual 
+like this: \[Edit Times\]
+
+## Playbacks
+
+![Diamond-H](/docs/images/Diamond-H.png)
+
+\(H\) The **Rotary Faders** at the top can be used to store
+a range of program functions including cues, chases, and fixture groups.
+The rotary faders have a level indicator around them which is shown in
+a user-configurable colour (halo colour).
+The touch screen below the faders shows legends indicating their function.
+
+\(I\) **The Page Select buttons** in the
+centre of the faders let you change the left or right fader banks to a different page.
+
+\(J\) Two more sets of **Rotary Faders** are provided above the Playback Faders.
+
+\(K\) The motorised **Playback faders** are the main place to store and play back cues or chases
+but can also be assigned a range of other functions. Each fader has a level indicator
+which can be set to user-configurable halo colour.
+The touch screen above the faders shows information about
+each playback.
+
+\(L\) The **Page Select buttons** let you change the left or right
+fader banks to a different page
+of playbacks (for both the playback faders and the lower rotary faders). 
+
+\(M\) The **Macro Buttons** store frequently used sequences of keypresses
+which can be replayed with a single button press. Playbacks can also be
+stored here. The display immediately above the buttons shows legends for
+the buttons. There is a smaller bank of 6 buttons between the playback faders with
+their own legend screen.
+
+\(N\) The **Playback controls** are used to control the playback of cue
+lists and include a configurable T-bar for the Scene Master function and motorised A-B faders.
```

Comment:
```suggestion
lists and include a T-bar to control the Scene Master and a motorised fader pair
in the Sequence Control Section to control the connected playback/master.
```

Reply:
Maybe also worth mentioning the scene master status screen

Reply:
Would you not call these A-B faders, that is how I would know them on other consoles, I suppose from the old 2-preset days.

Reply:
Mm... at release, they don't support separate In/Out A/B crossfading. The left fader is playback level and the right fader is manual crossfade (whole cue) or effect rate multiplier... the plan is for it to eventually support this but yeah it doesn't really support AB-type functionality as yet, sadly...

Reply:
Ah OK, I will use your description.